### PR TITLE
Fix finding of git remote branches

### PR DESCRIPTION
**Pull request by akorneta**
@skabashnyuk @sleshchenko @evoevodin 
Signed-off-by: Anton Korneta akorneta@codenvy.com

### CODE REVIEW

**Comment by voievodin** [comment]
The code looks ok.

**Comment by skabashnyuk** [comment]
ok

File: core/platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitProjectImporter.java [hunk 1]
```diff
@@ -338,15 +343,21 @@ private void cleanGit(File project) {
...
-                                                              .withListMode(BranchListRequest.LIST_REMOTE));
-        final Optional<Branch> first = remotes.stream()
-                                              .filter(br -> branchName.equals(br.getDisplayName()))
-                                              .findFirst();
-        if (!first.isPresent()) {
-            throw new GitException("Failed to get remote branch name", FAILED_CHECKOUT);
+    /**
+     * Looking for remote branch with specified name if it does not exist null  will be returned.
+     */
+    private String getRemoteBranch(GitConnection git, String branchName, String url) throws GitException {
```
Comment (sleshchenko) [inline core/platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitProjectImporter.java]: Rename getRemoteBranch to getRemoteBranchRef

**Comment by sleshchenko** [comment]
Other OK

**Pull request merged**